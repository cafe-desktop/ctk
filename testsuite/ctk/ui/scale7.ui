<!-- test invalid scale subelement -->
<interface>
  <object class="CtkScale">
    <marks>
    </marks>
    <foo/>
  </object>
</interface>
