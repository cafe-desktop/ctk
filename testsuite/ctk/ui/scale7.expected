ERROR: ctk-builder-error-quark 1
<input>:6:11 Unhandled tag: <foo>
